tsl/src/compression/segment_meta.c

```diff
@@ -144,3 +144,11 @@ segment_meta_min_max_builder_empty(SegmentMetaMinMaxBuilder *builder)
 {
 	return builder->empty;
 }
+
+void
+segment_meta_min_max_builder_gettype_info(SegmentMetaMinMaxBuilder *builder, int16 *typ_len,
```

Comment:
From codecov, it looks like this function is never called. Is this the case and can it then be removed?